src/error.rs

```diff
@@ -1,2 +1,57 @@
+use std::fmt;
+use std::fmt::Display;
+
 pub use failure::Error;
 pub type Result<T> = std::result::Result<T, Error>;
+
+#[derive(Debug)]
+struct SourceWrapper {
```

Comment:
I need to come in and modernize the error handling

Could you add a test under `tests/cmd` to verify we show the full failure message?  This way as things change, we won't break this

Reply:
An existing test (`tests/cmd/yaml_error.md`) hit this code path and was failing. I fixed the test, so you can now see the extra error message produced by this change.